dev-cpp/atkmm/atkmm-2.28.1.ebuild

```diff
@@ -2,7 +2,9 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
-inherit gnome.org meson multilib-minimal
+
+PYTHON_COMPAT=(python3_{6..9})
```

Comment:
I'd prefer `PYTHON_COMPAT=( python3_{6..9} )` (e.g. with spaces) since that style is used 4463 times in the tree vs 88 times without spaces.

Reply:
python3_6 has been removed in January.

Reply:
Pushed with changed whitespace and dropped 3.6 as `PYTHON_COMPAT=( python3_{6..7} )`